cmd/cloud-controller-manager/BUILD

```diff
@@ -1,26 +1,48 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+package(default_visibility = ["//visibility:public"])
+
+load(
+    "@io_bazel_rules_go//go:def.bzl",
+    "go_binary",
+    "go_library",
+)
+load("//vendor/k8s.io/component-base/version:def.bzl", "version_x_defs")
+
+go_binary(
+    name = "cloud-controller-manager",
+    embed = [":go_default_library"],
+    pure = "on",
+    x_defs = version_x_defs(),
+)
 
 go_library(
     name = "go_default_library",
     srcs = ["controller-manager.go"],
-    importpath = "k8s.io/cloud-provider-gcp/cmd/cloud-controller-manager",
-    visibility = ["//visibility:private"],
+    importpath = "k8s.io/kubernetes/cmd/cloud-controller-manager",
```

Comment:
Is this saying this package is the k8s.io/kubernetes CCM package?

Reply:
Fixed.

Reply:
thanks... I see a lot of other instances of this as well in other BUILD files